src/work-allocation/work-allocation-feature.routes.ts

```diff
@@ -1,14 +1,18 @@
 import { ModuleWithProviders } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { HealthCheckGuard } from 'src/app/shared/guards/health-check.guard';
+import { TaskHomeComponent } from './containers/task-home/task-home.component';
 import { TaskListComponent } from './containers/task-list/task-list.component';
 import { TaskManagerComponent } from './containers/task-manager/task-manager.component';
 
 export const ROUTES: Routes = [
     {
       path: '',
-      component: TaskListComponent
+      component: TaskHomeComponent
     },
+    // TODO: We probably don't need the following routes,
+    // as these components parents should be TaskHomeComponent.
+    // to discuss.
```

Comment:
`src/cases/case-feature.routes.ts` shows an example of this type of routing. I think the URL will still need to be distinct to differentiate between viewing the `TaskListComponent` and the `TaskManagerComponent`.

My suggestion would be to make the `task-list` route just `list` and the `task-manager` route just `manager` and then make both routes children of the default route (`path: ''`) so we get urls of `/tasks/list` and `/tasks/manager`.